shared/settings/password/index.tsx

```diff
@@ -139,6 +139,7 @@ class UpdatePassword extends Component<Props, State> {
               keyboardType={keyboardType}
               value={this.state.passwordConfirm}
               onChangeText={password => this._handlePasswordConfirmChange(password)}
+              onEnterKeyDown={() => this.props.onSave(this.state.password, this.state.passwordConfirm)}
```

Comment:
make sure this is disabled at the same time the save button is

Reply:
bleh i thought onsave checked it, it blocked on the rpcs instead